app/views/albums/new.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class="albums__topNav">
+  <%= button_to 'Home', root_path, method: 'get', class: "albums__homeButton" %>
+</div>
+
+<% if @album.errors.messages.present? %>
+  <%= render partial: "errors", locals: { record: @album } %>
+<% end %>
+
+<%= form_for @album do |f| %>
+  <table class="albums__table">
+    <col style="width:50%">
+    <col style="width:50%">
+    <tr class="albums__tableHeader">
+      <th colspan=2>
+        <h2>Create New Record</h2>
+      </th>
+    </tr>
+    <tr class="albums__tableRow">
+      <th>
+        <%= f.label :title %>
+      </th>
```

Comment:
notice how these forms are so similar? They're almost exactly the same. Refactor this form to be a partial